### PR TITLE
Completed the Event Summary Section

### PR DESCRIPTION
**Pull request by Matthewokiemute**
I was able to setup the StoryBlok
But I couldn't make use of at the section component 

I think I finally understand how to remove the .yarn folder now
I was running "git add ." and that was why everything was coming

### CODE REVIEW

**Review by 0xJamin — CHANGES_REQUESTED** [review]
Remove `.yarn` folder

File: public/images/cm8.png
Comment (0xJamin) [inline public/images/cm8.png:1]: This image is already in the CMS no need for it here

File: types.d.ts [hunk 1]
```diff
@@ -25,10 +25,16 @@ type HeroType = StoryblokBlok<"section_hero"> & {
   description: string;
   cta_buttons: ElementLinkType[];
 };
+//Storyblok EventSummary Section
```
Comment (0xJamin) [inline types.d.ts:28]: This comment is not needed

File: types.d.ts [hunk 2]
```diff
@@ -25,10 +25,16 @@ type HeroType = StoryblokBlok<"section_hero"> & {
...
+type EventSummaryType = StoryblokBlok<"section_event_summary"> & {
```
Comment (0xJamin) [inline types.d.ts:29]: This data structure is incorrect, please check the CMS for the correct one and describe the data accordingly

File: yarn.lock
Comment (0xJamin) [inline yarn.lock:1]: why does your `yarn.lock` file has so many changes?

**Review by 0xJamin — CHANGES_REQUESTED** [review]

**Comment by 0xJamin** [comment]
@Matthewokiemute why can't you use the data from Storyblok?🤔

File: yarn.lock
Comment (Matthewokiemute) [reply yarn.lock:1]: I don't have any idea about that

It updated when I ran "yarn install"

File: components/sections/EventSummary/EventSummary.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import React from "react";
+import cm8 from "../../../public/images/cm8.png";
+import cm6 from "../../../public/images/cm6.png";
+import cm1 from "../../../public/images/cm1.png";
+
+const EventSummary = ({ blok }: { blok: EventSummaryType }) => {
+  const { topics, sponsors, summary } = blok;
```
[the remark this answers is not in the document]
Comment (Matthewokiemute) [reply components/sections/EventSummary/EventSummary.tsx:8]: I tried it nothing will show as the content

That's why I removed them

Maybe if I add them you are the one to connect the contents to the from the StoryBlok CMS

File: components/sections/EventSummary/EventSummary.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      <div className="col-start-1 row-start-2 col-span-3 md:col-span-4 md:row-start-1 md:row-span-2 md:overflow-hidden">
+        <Image
+          src={cm8}
+          className="object-cover w-full h-full"
+          alt="Previous event summary image"
+        />
+      </div>
+      <div className="bg-[#181B36] text-white flex flex-col text-center justify-center md:static">
+        <p className="text-3xl">20+</p>
+        <p className="ml-6 text-lg">Sponsors</p>
```
[the remark this answers is not in the document]
Comment (Matthewokiemute) [reply components/sections/EventSummary/EventSummary.tsx:20]: Okay Changed them

File: .yarnrc.yml [hunk 5]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
[the remark this answers is not in the document]
Comment (Matthewokiemute) [reply .yarnrc.yml:1]: Okay sir

File: components/sections/EventSummary/EventSummary.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (0xJamin) [reply components/sections/EventSummary/EventSummary.tsx:8]: Screenshot 2023-07-11 at 2 05 50 PM
@Matthewokiemute  Just checked your component and see the data received as props, you mean you can't display this data on the UI?

File: components/sections/EventSummary/EventSummary.tsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import cm6 from "../../../public/images/cm6.png";
+import cm1 from "../../../public/images/cm1.png";
+
+const EventSummary = ({ blok }: { blok: EventSummaryType }) => {
+  const { topics, sponsors, summary } = blok;
+  return (
+    <section className="w-full grid grid-cols-2 grid-rows-3 md:grid-cols-6 md:grid-rows-3 mt-16 h-[800px]">
+      <div className="col-start-1 row-start-2 col-span-3 md:col-span-4 md:row-start-1 md:row-span-2 md:overflow-hidden">
+        <Image
+          src={cm8}
```
Comment (0xJamin) [inline components/sections/EventSummary/EventSummary.tsx:13]: Images as well are also on the CMS, please use them too

**Review by 0xJamin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**